requirements.txt

```diff
@@ -2,9 +2,9 @@
 numpy>=1.17
 astropy
 matplotlib
-lenstronomy
-galsim
 git+git://github.com/LSSTDESC/gcr-catalogs#egg=GCRCatalogs
+git+git://github.com/sibirrer/lenstronomy#egg=lenstronomy
```

Comment:
on the testing built failing: it looks like it wants to generate a conda environment and the cause of the failure is most likely that I have not registered a conda version of lenstronomy. Also note: there is a second requirement that does not have a direct conda version. If it is not necessary to have a conda built but instead go for the regular python built (you could e.g. look at the testing configuration of the skeleton package on the LSST-strong-lensing repo, or the lenstronomy source code)

Reply:
also, fyi, I just had troubles in lenstronomy as well to provide installations via git+git:// commands as it has some security guards in place recently.

Reply:
Good to know. Hopefully, Heather's env will fix this. The issue is that we need an env here to have the stack running.